docs/src/further_topics/ugrid/partner_packages.rst

```diff
@@ -0,0 +1,98 @@
+.. _ugrid partners:
+
+Iris' Mesh Partner Packages
+****************************
+Python is an easy to use language and has formed a very strong collaborative
+scientific community, which is why Iris is written in Python. But it is not as
+computationally strong as other languages, and this is a problem for working
+with unstructured meshes where computations are higher in volume and complexity
+(:ref:`see the data model detail <ugrid implications>`). So there are
```

Comment:
I'm not sure I'm comfortable saying this about Python - naively written Python is slow, and calls down to C are a way that Python combines speed with convenience. I'd say we're continuing a known strength of Python rather than working around a weakness (though I guess people working with meshes are probably fairly sold on Python so maybe I don't need to worry). I'm happy to write a variant of this if that would be useful

Reply:
a31546293bde29313aa7d24dd647ac27476c6604